layouts/partials/sidebar-tree.html

```diff
@@ -47,6 +47,14 @@
   {{ $show := cond (or (lt $ulNr $ulShow) $activePath (and (not $shouldDelayActive) (eq $s.Parent $p.Parent)) (and (not $shouldDelayActive) (eq $s.Parent $p)) (and (not $shouldDelayActive) ($p.IsDescendant $s.Parent))) true false -}}
   {{ $mid := printf "m-%s" ($s.RelPermalink | anchorize) -}}
   {{ $pages_tmp := where (union $s.Pages $s.Sections).ByWeight ".Params.toc_hide" "!=" true -}}
+  {{ with site.Params.language_alternatives -}}
```

Comment:
Consider adding a Go template comment on the previous line, to explain the intent here.

Reply:
added in a comment. good suggestion, thanks @sftim!